plugin/pkg/admission/podnodeselector/admission.go

```diff
@@ -259,3 +259,20 @@ func (p *Plugin) getNodeSelectorMap(namespace *corev1.Namespace) (labels.Set, er
 	}
 	return selector, nil
 }
+
+func isSubset(subSet, superSet labels.Set) bool {
+	if len(subSet) == 0 {
```

Comment:
looks like this logic was reversed.  While I understand that the previous behavior was wrong, we cannot change the meaning of the empty superset.